triagebot.toml

```
@@ -8,3 +8,14 @@ allow-unauthenticated = [
 ]
 
 [assign]
+
+[ping.icebreaker-llvm]
```

```suggestion
[ping.icebreakers-llvm]
```

needs to match https://team-api.infra.rust-lang.org/v1/teams.json